src/malli/swagger.cljc

```diff
@@ -22,6 +21,25 @@
 
 (defmethod accept :tuple [_ _ children _] {:type "array" :items {} :x-items children})
 
+;; Number formats are only defined in Swagger/OpenAPI spec.
+
+(defmethod accept 'number? [_ _ _ _] {:type "number" :format "double"})
+(defmethod accept 'integer? [_ _ _ _] {:type "integer" :format "int32"})
+(defmethod accept 'int? [_ _ _ _] {:type "integer" :format "int64"})
+(defmethod accept 'pos-int? [_ _ _ _] {:type "integer", :format "int64", :minimum 1})
+(defmethod accept 'neg-int? [_ _ _ _] {:type "integer", :format "int64", :maximum -1})
+(defmethod accept 'nat-int? [_ _ _ _] {:type "integer", :format "int64" :minimum 0})
+(defmethod accept 'float? [_ _ _ _] {:type "number" :format "float"})
+(defmethod accept 'double? [_ _ _ _] {:type "number" :format "double"})
+
+(defmethod accept :int [_ schema _ _]
+  (merge {:type "integer" :format "int64"}
+         (-> schema m/properties (select-keys [:min :max]) (set/rename-keys {:min :minimum, :max :maximum}))))
```

Comment:
Just double-check: is this inclusive or exclusive? Json schema separates the two.

Reply:
Inclusive.